### PR TITLE
#159330993 Completes sequelize and database setup

### PR DESCRIPTION
**Pull request by chukwuemekachm**
#### What does this PR do?
- Completes sequelize and database setup
#### Description of Task to be completed?
- Install sequelize
- Install pg
- Install pg-hstore
- Install sequelize-cli
- Install dotenv
- Setup sequelize
- Add .env-sample file to show the environmental variables used
- Add migrate and migrate:undo scripts to package.json
#### How should this be manually tested?
- Clone the repo
- Start command line
- Change to the project directory
- Run `npm install`
- Run `npm run migrate`
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #159330993
#### Screenshots (if appropriate)
- None
#### Questions:
- None

### CODE REVIEW

File: server/config/config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
```
Comment (paradisekelechi) [inline server/config/config.js:1]: Why not use es6 imports?

File: server/config/config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = {
```
Comment (paradisekelechi) [inline server/config/config.js:3]: Why not use es6 exports?

File: server/config/config.js (hunk 1 above)
Comment (chukwuemekachm) [reply server/config/config.js:1]: I'm having issues using es6 imports and exports from sequelize

File: server/config/config.js (hunk 2 above)
Comment (chukwuemekachm) [reply server/config/config.js:3]: Sequelize does not support the es6 imports and exports.

**Pull request merged**